express.js

```diff
@@ -245,7 +247,16 @@ setTimeout(() => {
 	} catch (e) {
 		console.error("Error while updating leaderboards!", e);
 	}
-}, 60 * 1000);
+},  parseInt(process.env.LEADERBOARDS_UPDATE_INTERVAL) * 1000);
+
+// Run recent players detection every 15 minutes
+setTimeout(() => {
```

Comment:
Please check #196 as axel1200 fixes the broken intervalls.

Reply:
I'm tempted to close his pull request. He is adding a new useless dependency to do something that JS already does, and he doesnt move the intervals to constant, so I do not see the point.

Reply:
The point is the intervals do not work in the current website and nobody so far could tell my why.

Reply:
Then this is a problem to look into. `setInterval` is, according to w3c docs, not supposed to be broken. Adding a new dependency ontop of it without understanding the underlying problem is not the best of ideas.

Reply:
I'll look into it once i'm done with this PR and the other